From 3f1dba9a821b53b42001605f9a126a958804884f Mon Sep 17 00:00:00 2001
From: Takashi Iwai <tiwai@suse.de>
Date: Mon, 9 Nov 2015 13:37:26 +0100
Subject: [PATCH] topology: Add missing include sys/stat.h

Necessary for proper definitions of S_IRUSR & co. Otherwise it
results in compile errors with old glibc and uclibc:
  parser.c: In function 'snd_tplg_build_file':
  parser.c:262: error: 'S_IRUSR' undeclared (first use in this function)
  parser.c:262: error: (Each undeclared identifier is reported only once
  parser.c:262: error: for each function it appears in.)

Upstream-Status: Backport

Signed-off-by: Takashi Iwai <tiwai@suse.de>
Signed-off-by: Maxin B. John <maxin.john@intel.com>
---
 src/topology/parser.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/topology/parser.c b/src/topology/parser.c
index 80a0ae0..18bb9c7 100644
--- a/src/topology/parser.c
+++ b/src/topology/parser.c
@@ -16,6 +16,7 @@
            Liam Girdwood <liam.r.girdwood@linux.intel.com>
 */
 
+#include <sys/stat.h>
 #include "list.h"
 #include "tplg_local.h"
 
-- 
2.4.0

